build/setup.d/100-install-open-policy-agent.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -euo pipefail
+
+curl --fail -L -o opa https://github.com/open-policy-agent/opa/releases/download/v0.16.2/opa_linux_amd64
+mv opa /opt/taupage/bin
```

Comment:
Can you verify the checksum of the downloaded file before proceeding?

Reply:
Can you please share an example

Reply:
https://github.com/zalando-stups/taupage/blob/master/build/setup.d/19-install-ec2-instance-connect.sh